flink-libraries/flink-table/src/main/scala/org/apache/flink/table/runtime/aggregate/AggregateUtil.scala

```diff
@@ -916,6 +916,10 @@ object AggregateUtil {
                   new DecimalMinAggFunction
                 case BOOLEAN =>
                   new BooleanMinAggFunction
+                case VARCHAR =>
+                  new StringMinAggFunction
+                case CHAR =>
```

Comment:
I do not think we are currently support CHAR type. For instance, I think if you try 'CHAR.max, you will see problems.